src/command/http-command.ts

```diff
@@ -391,6 +401,11 @@ export class HttpCommand implements CommandInterface<HttpOptions> {
 					...cert.subject,
 					alt: cert.subjectaltname,
 				},
+				serialNumber: cert.serialNumber.match(/.{2}/g)!.join(':'),
+				fingerprint256: cert.fingerprint256,
+				keyType: cert.asn1Curve || cert.nistCurve ? 'EC' : cert.modulus || cert.exponent ? 'RSA' : null,
```

Comment:
@alexey-yarmosh just fyi, `getX509Certificate()` is indeed for server certificates, but there's also `getPeerX509Certificate()` which does work for our case. But it returns some other fields in a form that we would have to additionally parse, so I decided to keep your approach. ESA and EC are the only two types we should encounter on the web so it should be fine.